content/en/news/releases/1.13.x/announcing-1.13/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.12
+linktitle: 1.12
+subtitle: Major Update
+description: Istio 1.12 release announcement.
+publishdate: 2021-11-18
+release: 1.12.0
+skip_list: true
+aliases:
+    - /news/announcing-1.12
+    - /news/announcing-1.12.0
+---
+
+We are pleased to announce the release of Istio 1.13!
+
+{{< tip >}}
+Istio 1.13.0 is officially supported on Kubernetes versions `1.19` to `1.23`.
+{{< /tip >}}
+
+
+Here are some of the highlights of the release:
+
+
+
+- **Added** support for hostname-based multi-network gateways for east-west traffic. The hostname will be resolved in
```

Comment:
I think we generally write this page in paragraph format

Reply:
Considering the entire PR uses bullet items and the release timeline, I suggest we keep the bullet items.